docs/src/modules/components/AppNavDrawer.js

```diff
@@ -54,6 +55,15 @@ function ProductDrawerButton(props) {
     setAnchorEl(null);
   };
 
+  const handleEventDelegation = (e) => {
+    // Assert whether an 'a' tag resides in the parent of the clicked element through which the event bubbles out.
+    const isLinkInParentTree = e?.target ? Boolean(e.target.closest('a')) : false;
```

Comment:
In which case would you get  the event undefined?
```suggestion
    const isLinkInParentTree = Boolean(event.target.closest('a')) ;
```

Reply:
Makes sense, I was checking the event handler type and thought that target is optional there but it wasn't. Done the other change too.

Reply:
Makes sense, I was checking the event handler type and thought that target is optional there but it wasn't. Done the other change too.